### PR TITLE
adapt guides to fixed code blocks format

### PR DESCRIPTION
**Pull request by erciccione**
Requires #379

Fixed use of backticks on all user guides which use them.

### CODE REVIEW

File: guide/masternode-setup.md [hunk 1]
```diff
@@ -174,82 +184,101 @@ You will also see prompts for user information, but this can be left blank.
...
+```
+apt install ufw
+```
  
 (press Y and Enter to confirm) 
 
 The next step opens port 8168 which is required for your masternode to communicate.
+
+```
+ufw allow ssh/tcp
```
[the remark this answers is not in the document]
Comment (erciccione) [reply guide/masternode-setup.md:214]: We could add a line break to make clear it's different commands, but my feeling is that if somebody launch them all together, they probably have close to 0 familiarity with the command line, so it's hard to come up with something visually clearer. We could add a `$` before the commands like some do, but that's not good for copy pasting (unless we add a specific "copy" button, but i'd say that's very excessive)

Maybe the most effective approach would be to specifically tell people right before the code block that each command should be run by itself. Thoughts?

[the remark this answers is not in the document]
Comment (justanwar) [reply guide/masternode-setup.md:214]: Perhaps keep the formatting for the parts that needs to be executed separately as it is?

[the remark this answers is not in the document]
Comment (erciccione) [reply guide/masternode-setup.md:214]: Something like this? The fixed format removes a lot of space between blocks, so it should look clear enough for people to understand the different commands, but doesn't clutter the page.

Screenshot_20240418_095350

[the remark this answers is not in the document]
Comment (justanwar) [reply guide/masternode-setup.md:214]: Yes, this works.

**Review by justanwar — APPROVED** [review]

File: guide/masternode-setup.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (erciccione) [reply guide/masternode-setup.md:216]: Pushed the change.

**Review by justanwar — APPROVED** [review]

**Pull request merged**